episodes/08-branches.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Branches
+teaching: 5
+exercises: 0
+---
+
+::::::::::::::::::::::::::::::::::::::: objectives
+
+- What are branches?
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+:::::::::::::::::::::::::::::::::::::::: questions
+
+- Understand what branches are and when to use them
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+::: instructor
+Explain what branches are and when to use them, we will practice with the git commands in the next episode.
+:::
+
+Have a look at 
+[these slides introducing branches](https://esciencecenter-digital-skills.github.io/digital-skills-slides/modules/git-lesson/branches).
+
+
```

Comment:
I think it would be good to explain here at least briefly git switch, git branch, git checkout (there are always people who ask about it). And add an exercise about it as in e.g. https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging??

Reply:
We decided to add a few basic commands that are often used. But considering we practice with this in episode 9 we did not add exercises.